db-scripts/src/main/resources/migration.sql

```diff
@@ -425,4 +426,31 @@ INSERT INTO reference_genome_gene (ENTREZ_GENE_ID, CYTOBAND, EXONIC_LENGTH, CHR,
 FROM `gene`);
 
 UPDATE info SET DB_SCHEMA_VERSION="2.4.1";
--- ========================= end of reference genes related tables ========================================================================
+-- ========================= end of reference genes related tables =====================================================
+
+##version: 2.4.2
+-- ================== new reference genome foreign key in mutation_event and genetic_profile tables   ==================
+ALTER TABLE `mutation_event` ADD COLUMN `REFERENCE_GENOME_ID` INT(4) NULL AFTER `ENTREZ_GENE_ID`;
+ALTER TABLE `mutation_event`
+    ADD FOREIGN KEY (`REFERENCE_GENOME_ID`) REFERENCES `reference_genome` (`REFERENCE_GENOME_ID`) ON DELETE CASCADE;
+
+UPDATE `mutation_event` set REFERENCE_GENOME_ID = 1 WHERE NCBI_BUILD in ('hg19', 'GRCh37','37');
```

Comment:
you shouldn't assume 1, 2 ,3 since the `REFERENCE_GENOME_ID` is auto incremented. You can get the ID using a select statement.

Reply:
@pieterlukasse , good idea, change made, thanks!